packages/autocertifier-client/src/data/Certificate.ts

```diff
@@ -0,0 +1,4 @@
+export interface Certificate {
```

Comment:
What information this class contains? Maybe a certificate and the corresponding key?

```suggestion
export interface CertificateAndKey {
```